demos/python_demos/instance_segmentation_demo/README.md

```diff
@@ -1,6 +1,7 @@
 # Instance Segmentation Python* Demo
 
 This demo shows how to run Instance Segmentation models from `Detectron` or `maskrcnn-benchmark` using OpenVINO&trade;.
+These models should be obtained through [OpenVINO&trade; Training Extensions (OTE)](https://github.com/opencv/openvino_training_extensions).
```

Comment:
I think leaving https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/instance_segmentation#get-pretrained-models link is more useful.